Panic in the portlayer along the attach path

[Test-Cases.Group1-Docker-Commands.1-09-Docker-Attach-VCH-13853-3107-container-logs.zip](https://github.com/vmware/vic/files/1341407/Test-Cases.Group1-Docker-Commands.1-09-Docker-Attach-VCH-13853-3107-container-logs.zip)

```
Sep 27 2017 21:33:18.014Z DEBUG [ END ] [github.com/vmware/vic/lib/portlayer/attach/communication.(*Connector).reqs:411] [304.578682ms] 
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xd06f28]

goroutine 140 [running]:
github.com/vmware/vic/lib/portlayer/attach/communication.(*LazySessionInteractor).SessionInteractor(0x0, 0x0, 0x0)
	/go/src/github.com/vmware/vic/lib/portlayer/attach/communication/lazy.go:56 +0x38
github.com/vmware/vic/lib/portlayer/attach/communication.(*Connector).RemoveInteraction(0xc420d6aa80, 0xc420d6d1c0, 0x40, 0x0, 0x0)
	/go/src/github.com/vmware/vic/lib/portlayer/attach/communication/connector.go:211 +0xfb
github.com/vmware/vic/lib/portlayer/attach/communication.(*Connector).reqs(0xc420d6aa80, 0xc420bbbf20, 0x1db8a40, 0xc4208bd200, 0xc420bdb450, 0x1, 0x1)
	/go/src/github.com/vmware/vic/lib/portlayer/attach/communication/connector.go:468 +0x77e
created by github.com/vmware/vic/lib/portlayer/attach/communication.(*Connector).processIncoming
	/go/src/github.com/vmware/vic/lib/portlayer/attach/communication/connector.go:348 +0x7e8
time="2017-09-27T21:33:22Z" level=info msg="Launching portlayer server pprof server on 127.0.0.1:6063" 
```


[Test-Cases.Group1-Docker-Commands.1-09-Docker-Attach-VCH-13853-3107-container-logs.zip](https://github.com/vmware/vic/files/1341408/Test-Cases.Group1-Docker-Commands.1-09-Docker-Attach-VCH-13853-3107-container-logs.zip)

--- a/lib/portlayer/attach/communication/connector.go
+++ b/lib/portlayer/attach/communication/connector.go
@@ -208,6 +208,11 @@ func (c *Connector) RemoveInteraction(id string) error {
 	}
 	c.mutex.Unlock()
 
+	// the !ok case, but let's check the actual condition that impacts us
+	if v == nil {
+		return nil
+	}
+
 	conn := v.SessionInteractor()
 	if conn != nil {
 		err = conn.Close()

